Services/UI/OJS.Services.Ui.Data/Implementations/ContestsDataService.cs

```diff
@@ -137,6 +137,9 @@ public IQueryable<Contest> GetLatestForParticipantByUsername(string username)
                  .ThenInclude(pg => pg.Problems)
                     .ThenInclude(p => p.SubmissionTypesInProblems)
                         .ThenInclude(sp => sp.SubmissionType)
+            .Include(c => c.ProblemGroups)
+                .ThenInclude(pg => pg.Problems)
+                    .ThenInclude(p => p.Checker)
```

Comment:
@shsimeonova can you check if we will have performance boost if we use `.AsSplitQuery` on this query.
We should aim to use service model in the first place, not get all the data, but I assume this will be more difficult to implement right now..